hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancer.java

```diff
@@ -459,6 +461,22 @@ private void validateConfiguration(ContainerBalancerConfiguration conf)
           "hdds.container.balancer.move.replication.timeout should " +
           "be less than hdds.container.balancer.move.timeout.");
     }
+
+    // (move.timeout - move.replication.timeout - event.timeout.datanode.offset)
+    // should be greater than or equal to 9 minutes
+    long datanodeOffset = ozoneConfiguration.getTimeDuration("hdds.scm.replication.event.timeout.datanode.offset",
+        Duration.ofMinutes(6).toMillis(), TimeUnit.MILLISECONDS);
+    if ((conf.getMoveTimeout().toMillis() - conf.getMoveReplicationTimeout().toMillis() - datanodeOffset)
+        < Duration.ofMinutes(9).toMillis()) {
+      String msg = String.format("(hdds.container.balancer.move.timeout (%sm) - " +
+              "hdds.container.balancer.move.replication.timeout (%sm) - " +
+              "hdds.scm.replication.event.timeout.datanode.offset (%sm)) should be greater than or equal to 9 minutes.",
```

Comment:
Small suggestion, it's better to just the log the duration in milliseconds so we know precisely how much it is when parsing the logs.